### ISSUE TITLE
Pre-trained model performing badly on MPI-INF 3D HP

### ISSUE THREAD

**Issue by NiteshBharadwaj** [xingyizhou/pytorch-pose-hg-3d#40]
The pre-trained model linked here is not giving the reported accuracy for MPI-INF. (200 MPJPE on test set)
But the pre-trained model for lua is working as expected (124 MPJPE on test set)

3D regressor isn't working as expected
